cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/LaunchLoadBalancerHandler.java

```diff
@@ -0,0 +1,116 @@
+import com.sequenceiq.cloudbreak.cloud.model.CloudResourceStatus;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+import com.sequenceiq.cloudbreak.cloud.notification.PersistenceNotifier;
+import com.sequenceiq.cloudbreak.cloud.scheduler.SyncPollingScheduler;
+import com.sequenceiq.cloudbreak.cloud.task.PollTask;
+import com.sequenceiq.cloudbreak.cloud.task.PollTaskFactory;
+import com.sequenceiq.cloudbreak.cloud.task.ResourcesStatePollerResult;
+import com.sequenceiq.cloudbreak.cloud.transform.ResourceLists;
+import com.sequenceiq.cloudbreak.cloud.transform.ResourcesStatePollerResults;
+
+import reactor.bus.Event;
+import reactor.bus.EventBus;
+
+@Component
+public class LaunchLoadBalancerHandler implements CloudPlatformEventHandler<LaunchLoadBalancerRequest> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LaunchLoadBalancerHandler.class);
+
+    @Inject
+    private CloudPlatformConnectors cloudPlatformConnectors;
+
+    @Inject
+    private SyncPollingScheduler<ResourcesStatePollerResult> syncPollingScheduler;
+
+    @Inject
+    private PollTaskFactory statusCheckFactory;
+
+    @Inject
+    private PersistenceNotifier persistenceNotifier;
+
+    @Inject
+    private EventBus eventBus;
+
+    @Override
+    public Class<LaunchLoadBalancerRequest> type() {
+        return LaunchLoadBalancerRequest.class;
+    }
+
+    @Override
+    public void accept(Event<LaunchLoadBalancerRequest> launchLoadBalancerRequestEvent) {
+        LOGGER.debug("Received event: {}", launchLoadBalancerRequestEvent);
+        LaunchLoadBalancerRequest request = launchLoadBalancerRequestEvent.getData();
+        CloudContext cloudContext = request.getCloudContext();
+        CloudStack cloudStack = request.getCloudStack();
+        CloudConnector<Object> connector = cloudPlatformConnectors.get(cloudContext.getPlatformVariant());
+        AuthenticatedContext ac = connector.authentication().authenticate(cloudContext, request.getCloudCredential());
+        try {
+            LaunchLoadBalancerResult result;
+            if (!cloudStack.getLoadBalancers().isEmpty()) {
+                ResourcesStatePollerResult statePollerResult = launchLoadBalancers(cloudStack, connector, ac,
+                    persistenceNotifier, cloudContext);
+                result = ResourcesStatePollerResults.transformToLaunchLoadBalancerResult(request, statePollerResult);
+                request.getResult().onNext(result);
+                LOGGER.debug("Launching the load balancers successfully finished for {}", cloudContext);
+            } else {
+                LOGGER.debug("No load balancers configured for stack.");
+                result = new LaunchLoadBalancerResult(request.getResourceId(), List.of());
+            }
+            eventBus.notify(result.selector(), new Event<>(launchLoadBalancerRequestEvent.getHeaders(), result));
+        } catch (Exception e) {
+            if (ExceptionUtils.getRootCause(e) instanceof InterruptedException) {
```

Comment:
Have you seen such issue? I think we could add these to everywhere, so either we should not add it here or handle it generally.
@sodre90 what do you think?

Reply:
We did some improvements, so now it should not cause any problem. Eventbus is stopped as the first step in graceful shutdown so this message will not be consumed.
Anyway as I see it does not do anything, just logging, but it will send the message to the event bus :) So we can leave it as it is.

Reply:
I haven't personally seen this myself. It was part of the `LaunchStackHandler`, and since the load balancer launch logic go through a very similar path through the cloud resource connectors as the stack launch logic does, I thought it was possible to run into this exception and that I should handle it.

Reply:
@sodre90 It sounds like this might not be necessary in either case anymore? If so I'll clean it up next time I'm in that part of the code.

Reply:
no, you don't need this anywhere.
also there is no `else` branch here, so this code has no effect other than logging. I would say it should be deleted just to clean it up